### DISCUSSION TITLE
Chromium cannot preview files

### DISCUSSION THREAD

**Discussion by dmavrommatis** [facelessuser/MarkdownPreview#120] [Q&A] [answered]
This is the same issue as #115 but I have more context on the reason this happens.

To avoid security concerns the `/tmp` directory is not available to programs installed with snap. This was thought to be a bug but it isn't: https://bugs.launchpad.net/ubuntu/+source/chromium-browser/+bug/1849437. 

This means that Chromium cannot actually read the temp files that this plugin is depending on. The snap temp directory for Chromium is located here instead:
```
/tmp/snap.chromium/tmp
```

Would it make sense to allow configuring the location of the temporary files so we can avoid these issues?

**Comment by facelessuser** [comment]
We can possibly look into this.

**Comment by facelessuser** [comment, marked as answer]
@dmavrommatis This is probably what you are looking for: https://facelessuser.github.io/MarkdownPreview/usage/#preview-temp-location.

**Comment by facelessuser** [comment]
As this is already implemented, and this is not a bug or feature request, moving this to discussions.
